test/e2e/suite/command/sign.go

```diff
@@ -150,4 +150,24 @@ var _ = Describe("notation sign", func() {
 				MatchErrContent(expectedErrMsg)
 		})
 	})
+
+	It("with TLS by digest", func() {
+		HostInGithubAction(BaseOptions(), func(notation *utils.ExecOpts, artifact *Artifact, vhost *utils.VirtualHost) {
+			notation.Exec("sign", "-d", artifact.DomainReferenceWithDigest()).
```

Comment:
Just curious, why do we need to test with the `-d` flag enabled?

Reply:
To print the request log for ensuring the connection is HTTP or HTTPS